content/docs/user-guide/packaging/project-export/project-export-ios.md

```diff
@@ -0,0 +1,101 @@
+Assuming you are just following along using the O3DE Atom Sampleviewer project, this single invocation should be all you need to generate the necessary Xcode project file in the project's build folder:
+```
+export O3DE_ENGINE_PATH=/path/to/o3de
+$(O3DE_ENGINE_PATH)/scripts/o3de.sh export-project -es $(O3DE_ENGINE_PATH)/scripts/o3de/ExportScripts/export_source_ios_xcode.py -pp /path/to/o3de-atom-sampleviewer
+```
+Where `O3DE_ENGINE_PATH` is an environment variable that should point to the folder location of the O3DE Source engine on your machine.
+
+Assuming the underlying CMake build system finds no issues with your engine or project installation, you should have the corresponding Xcode project file, which will be located at `<PROJECT>/build/game_ios`. For the Atom SampleViewer, it should be called `AtomSampleViewer.xcodeproj`.
+
+### Using Xcode
+Double click the file `AtomSampleViewer.xcodeproj` to open it in Xcode. This should load everything needed to build the app on an iOS device.
+
+Deploying to an iOS device requires code-signing (even test builds). Make sure that you have a provisioning profile setup for code-signing.
+
+Next, we'll open the project configuration details. Click on the AtomSampleViewer project icon in the Explorer tree-view (left hand side of the Xcode IDE). On the left column under "General", select the build configuration `AtomSampleViewer.GameLauncher`. The resulting configuration page should look like this:
+
+{{< image-width "/images/user-guide/packaging/project-export-ios/atom-sampleviewer-configuration-page.png" "700">}}
+
+Then go to the "signing and capabilities" tab, and setup the provisioning profile to link to your AppleID. Customize the bundle identifier (add any string you want, for example: append the phrase 'test'). Enable "Automatically manage signing."
+
+{{< image-width "/images/user-guide/packaging/project-export-ios/signing-and-capabilities-page.png" "700">}}
+
+Now click on the bar in the region labeled “AtomSampleViewer.GameLauncher”, and scroll through the drop down to click “Edit Scheme.”
+
+{{< image-width "/images/user-guide/packaging/project-export-ios/edit-scheme.png" "700">}}
+
+Set the desired build configuration, and in the arguments section, set the relevant CLI parameters. An example configuration is shown below:
+
+{{< image-width "/images/user-guide/packaging/project-export-ios/scheme-configuration-debug-build.png" "700">}}
+{{< image-width "/images/user-guide/packaging/project-export-ios/scheme-configuration-cli-params.png" "700">}}
+
+{{< note >}}
+Subsequent runs of the iOS Export script will clear any CLI configurations you provide in the Xcode project file, because the entire project file is re-generated. If you need changes to stick, you can specify the commands to run on launch for a release build via code or by using the `autoexec.client.setreg`. like so: https://docs.o3de.org/docs/user-guide/packaging/windows-release-builds/#set-the-starting-level 
```

Comment:
```suggestion
Subsequent runs of the iOS Export script will clear any CLI configurations you provide in the Xcode project file, because the entire project file is re-generated. If you need changes to stick, you can specify the commands to run on launch for a release build via code or by using the `autoexec.client.setreg`. You can find an example of this in [Set the starting level](/docs/user-guide/packaging/windows-release-builds/#set-the-starting-level).
```